plugin/trino-hive/src/main/java/io/trino/plugin/hive/orc/OrcPageSource.java

```diff
@@ -245,6 +251,16 @@ public long getMemoryUsage()
         return memoryContext.getBytes();
     }
 
+    @Override
+    public Metrics getMetrics()
+    {
+        CompressionKind compressionKind = recordReader.getCompressionKind();
+        if (compressionKind.equals(NONE)) {
+            return EMPTY;
+        }
+        return new Metrics(ImmutableMap.of(compressionKind.name(), new LongCount(0)));
```

Comment:
The metric name can't be just the compression kind enum, as it doesn't say where the compression is encountered.
Maybe prefix the compression name with "OrcReaderCompressionFormat" and "ParquetReaderCompressionFormat"